### PR TITLE
Refactor intent email template

### PR DESCRIPTION
**Pull request by jrobbins**
I still need to add permissions checking.

### CODE REVIEW

File: processes.py [hunk 1]
```diff
@@ -50,8 +50,12 @@ def process_to_dict(process):
 # This page generates a preview of an email that can be sent
 # to a mailing list to announce an intent.
 # The parameter "{feature_id}" is filled in by JS code.
+# The param "intent" adds causes the template to include details
```
Comment (jpmedley) [inline processes.py:53]: "adds causes"?

File: templates/admin/features/launch.html [hunk 2]
```diff
@@ -32,18 +32,17 @@ <h3>Copy and send this text for your "Intent to ..." email</h3>
...
 <section>
 <h3>Obtain LGTMs from 3 API Owners</h3>
-<span class="help">Given the intent stage of this feature is "{{feature.intent_stage}}" that means you're now done with your experimentation and implementation, and have documented that in this feature description. Now you need to get 3 API Owners to respond to your message with LGTMs. Once you have that, you're cleared from the Blink Launch process perspective to ship your feature.
-</section>
-{% elif feature.intent_stage == "Implement and Ship" %}
-<section>
-<h3>Obtain LGTMs from 3 API Owners</h3>
-<span class="help">Given the intent stage of this feature is "{{feature.intent_stage}}" that means this is a small change and you’d like to ship enabled-by-default immediately. Per <a href="http://www.chromium.org/blink#launch-process">the launch process for new features</a>, you’ll now need 3 LGTMs from API OWNERS before shipping.
+<span class="help">
+  You will need to get 3 API Owners to respond to your message with LGTMs.
```
Comment (jpmedley) [inline templates/admin/features/launch.html:39]:
```suggestion
  You will need three LGTMs from API owners.
```

File: templates/admin/features/launch.html [hunk 3]
```diff
@@ -32,18 +32,17 @@ <h3>Copy and send this text for your "Intent to ..." email</h3>
...
+  Once you have that, you're cleared from the
+  <a href="http://www.chromium.org/blink#launch-process">Blink Launch process</a>
+  perspective to ship your feature.
```
Comment (jpmedley) [inline templates/admin/features/launch.html:42]:
```suggestion
According to the
<a href="http://www.chromium.org/blink#launch-process">Blink Launch process</a>
after that, you're free to ship your feature.
```

File: processes.py (hunk 1 above)
Comment (jrobbins) [reply processes.py:53]: I'll correct it to "clauses".

**Comment by jrobbins** [comment]
This should resolve issue #880.

**Comment by jrobbins** [comment]
@googlebot I fixed it.

**Comment by jrobbins** [comment]
Anthony: Sorry, somehow this CL picked up diffs to CSS and forms that are already on the main branch.  Please just review the changes to admin.py, launch_test.py, launch.html, and intent_to_implement.html.

**Pull request merged**